server/hedley/modules/custom/hedley_reports/scripts/generate-data-for-all.php

```diff
@@ -34,7 +34,7 @@
   ->addTag('exclude_deleted');
 
 if ($exclude_set) {
-  $base_query->addTag('exclude_set');
+  $base_query->addTag('exclude_set_reports_data');
```

Comment:
Where do we use this tag?

Reply:
@AronNovak 

At https://github.com/TIP-Global-Health/eheza-app/pull/1283/files#diff-576b4cc9568e1d723be865511b32f6fcb9147fdf926905d82686213e1435548dR23.

The context is that every encounter has to calculate it's completion data only once.
From that point on, it never changes.

So, the plan is to run the script once initially, and then, have overnight Jenkins task that will fill completion data for only those encounters, that don't have it set.